src/appengine.py

```diff
@@ -350,6 +351,11 @@ def convert_json_2() -> None:
     return convert_json(r"[A-Z]{2}(_\w+)?\/\w+.csv")
 
 
+@app.route("/errors_to_github")
+def errors_to_github() -> None:
+    return register_new_errors()
```

Comment:
Can we use a verb for the first token of the path? It could be `register_new_errors` or `report_errors_to_github` or ...

Reply:
Done.